draft-ietf-dnsop-svcb-https.md

```diff
@@ -1542,36 +1542,33 @@ This document defines a new DNS RR type, SVCB, whose value 64 has
 been allocated by IANA from the "Resource Record (RR) TYPEs"
 subregistry of the "Domain Name System (DNS) Parameters" registry:
 
-Type: SVCB
-
-Value: 64
-
-Meaning: General Purpose Service Endpoints
-
-Reference: This document
+* Type: SVCB
+* Value: 64
+* Meaning: General Purpose Service Binding
+* Reference: This document
 
 ## HTTPS RRType
 
 This document defines a new DNS RR type, HTTPS, whose value 65 has
 been allocated by IANA from the "Resource Record (RR) TYPEs"
 subregistry of the "Domain Name System (DNS) Parameters" registry:
 
-Type: HTTPS
-
-Value: 65
-
-Meaning: HTTPS Specific Service Endpoints
-
-Reference: This document
+* Type: HTTPS
+* Value: 65
+* Meaning: Service Binding for HTTPS
+* Reference: This document
 
 ## New registry for Service Parameters {#svcparamregistry}
 
-The "Service Binding (SVCB) Parameter Registry" defines the namespace
+IANA is requested to create a new registry, entitled
+"Service Parameter Keys (SvcParamKeys)".  This registry defines the namespace
 for parameters, including string representations and numeric
 SvcParamKey values.  This registry is shared with other SVCB-compatible
 RR types, such as the HTTPS RR.
 
-ACTION: create and include a reference to this registry.
+ACTION: create this registry, on a new page entitled
+"DNS Service Bindings (SVCB)" under the "Domain Name System (DNS) Parameters"
+category.
```

Comment:
[DNS Parameters[(https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml) doesn't currently appear to have any sub-pages, just a list of registries.  Requesting a new page, which suggests a collection of registries, under an existing collection of registries feels odd.  You pointed out that there is a sub-page of this collection (SSHFP), but it contains multiple registries.  (It's also kind of odd to me, since those registries are listed under DNS Parameters in the master list but aren't on the DNS Parameters page.)

Unless we anticipate adding more SVCB registries in the future, I would think this is a new registry under the DNS Parameters category.

Reply:
RR-type-specific registries that are on the main page:
* [AFSDB RR Subtype](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-8)
* [CSYNC Flags](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#csync-flags)
* [DHCID RR Identifier Type Codes](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-9)
* [DHCID RR Digest Type Codes](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dhcid-rr-digest-type-codes)
* [ZONEMD Schemes](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#zonemd-schemes)
* [ZONEMD Hash Algorithms](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#zonemd-hash-algorithms)

RR-type-specific registries that are on a different page within the category:
* [AMTRELAY Resource Record Parameters](https://www.iana.org/assignments/amtrelay-resource-record/amtrelay-resource-record.xhtml#relay-type-field)
* [DNS KEY Resource Record Protocol Octet Values](https://www.iana.org/assignments/dns-key-rr/dns-key-rr.xhtml#dns-key-rr-1)
* [DNSKEY FLAGS](https://www.iana.org/assignments/dnskey-flags/dnskey-flags.xhtml#dnskey-flags-1)
  - Separate page from the "Octet Values"!
* [DNS SSHFP Resource Record Parameters](https://www.iana.org/assignments/dns-sshfp-rr-parameters/dns-sshfp-rr-parameters.xhtml#dns-sshfp-rr-parameters-2)
  - Two registries on this page; the others only have one.

RR-type-specific registries that get their own category:
* [CERT RR Certificate Types](https://www.iana.org/assignments/cert-rr-types/cert-rr-types.xhtml)
* [DANE stuff](https://www.iana.org/assignments/dane-parameters/dane-parameters.xhtml#certificate-usages)
  - 3 registries about TLSA
* [Secret Key Transaction Authentication for DNS (TSIG) Algorithm Names](https://www.iana.org/assignments/tsig-algorithm-names/tsig-algorithm-names.xhtml#tsig-algorithm-names-1)

If there is an organizing principle here, it is highly mysterious, but if it were up to me, I would put registries for "DNS internal" types (e.g. DNSKEY, ZONEMD, CSYNC, TSIG) on the main page, and all the others (AFSDB, DHCID, DANE, AMTRELAY, SSHFP, CERT, SVCB) on separate pages within the category.

As a practical matter, adding a First-Come-First-Served registry to the (already enormous) main page of DNS parameters seems less than ideal.

Reply:
Fair enough -- I didn't realize there were so many.  It's a pity there isn't really a multi-tier option to make these more explorable.